### DISCUSSION TITLE
Syntax highlighting in Purebasic

### DISCUSSION THREAD

**Discussion by LinXP** [github-linguist/linguist#5855] [Q&A] [answered]
Hello everyone, is it possible to enable syntax highlighting from this repository?
https://github.com/peterthomashorn/purebasic-language-for-sublime-text
And what is needed for this?

**Comment by lildude** [comment, marked as answer]
Linguist already has PureBasic support so you only need to perform step 2 and 5 of [Adding a language](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language) to syntax highlighting.
